gepetuto/test.py

```diff
@@ -10,16 +10,30 @@
 
 def test(**kwargs):
     """Test python scripts."""
+    python_version = kwargs["p"]
+    if python_version == "":
+        python_version = retrieve_python_version()
     LOG.info("testing tutorial sources.")
     for tp_number in range(100):
         LOG.debug(f"Looking for tp {tp_number}")
         folder = Path(f"tp{tp_number}")
         if folder.exists():
             for python_file in folder.glob("*.py"):
                 LOG.debug(f"Checking {python_file}")
-                check_call([sys.executable, python_file])
+                check_call([python_version, python_file])
         else:
             if tp_number == 0:
                 continue
             break
     LOG.info("test passed.")
+
+
+def retrieve_python_version():
+    """Retrieve installed python version."""
+    version = sys.version_info[0]
+    if version == 3:
+        return "python3"
+    if version == 2:
+        return "python"
```

Comment:
I don't think we want to be compatible with python 2, but python 3 is sometimes called `python3`, sometimes called `python`, depending on the system

Reply:
if we have python 3, it's sometimes possible to call it with python, but is it always possible to call it with `python3` ?

Reply:
no, some systems have the `python3` command but no `python`, while other systems have the `python` command but no `python3`.